src/EventListener/AngularHtmlListener.php

```diff
@@ -81,9 +81,7 @@ private function getCompiledAssetFor(string $linked_file, File $owning_file, Rea
     {
         $file_path = $linked_file;
 
-        if ($file_path[0] === '.' && $owning_file->dir !== '.' && !empty($owning_file->dir)) {
-            $file_path = $owning_file->dir . '/' . $file_path;
-        }
+        $file_path = preg_replace('/\.css$/', '.less', $owning_file->dir . '/' . $file_path);
```

Comment:
This method is called for both style and for the template, so hardcoding anything with `.css` or `.less` doesn't feel right.